web-marketplace/src/components/molecules/CreditsAmount/CreditsAmount.mock.tsx

```diff
@@ -0,0 +1,25 @@
+export const creditVintages = [
+  {
+    date: 'Jan 1, 2019 - December 31, 2019',
+    credits: '100',
+  },
+  { date: 'Jan 1, 2020 - December 31, 2020', credits: '200' },
+  { date: 'Jan 1, 2021 - December 31, 2021', credits: '300' },
+];
+
+export const cryptoOptions = [
```

Comment:
I don't think this belongs to a mock file, this isn't data but static text and links so should probably be defined in some .constants.ts file or directly in the component, adding localization support with lingui

Reply:
I didn't realise this is static text. I'll update it accordingly.